### DISCUSSION TITLE
Adding a softmax layer or using the multi-class classification?

### DISCUSSION THREAD

**Discussion by stefanolipira** [ThilinaRajapakse/simpletransformers#1500] [General]
How is the classification model with labels=2 compiled? with a categorical cross-entropy loss function and categorical accuracy as the evaluation metric?
